packages/block-library/src/columns/variations.js

```diff
@@ -12,6 +12,28 @@ import { __ } from '@wordpress/i18n';
  * @type {WPBlockVariation[]}
  */
 const variations = [
+	{
+		name: 'one-column-full',
+		title: __( '100' ),
+		description: __( 'One column; full width' ),
+		icon: (
+			<SVG
+				width="48"
+				height="48"
+				viewBox="0 0 48 48"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<Path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="m39.0625 14h-30.0625v20.0938h30.0625zm-30.0625-2c-1.10457 0-2 .8954-2 2v20.0938c0 1.1045.89543 2 2 2h30.0625c1.1046 0 2-.8955 2-2v-20.0938c0-1.1046-.8954-2-2-2z"
+				/>
+			</SVG>
+		),
+		isDefault: true,
```

Comment:
I don't think there should be two variations with `isDefault: true`.

In practice this seems to get overridden by the `isDefault` on the two columns equal variation, so maybe lets remove this line.

Reply:
```suggestion
```

we'd like two columns as default, do we?

Reply:
Yup, I removed it. isDefault should be set to true for the two columns variation.